test/elixir_google_drive_api/drive_test.exs

```diff
@@ -58,4 +58,34 @@ defmodule ElixirGoogleDriveApi.DriveTest do
       end
     end
   end
+
+  describe "export_file" do
+    @headers [
+      "Content-Type": "application/json",
+      "Authorization": "Bearer ya29.Glv3BDWcd49KSskqCfSKgd_DURWF7VAbtr_b_tLneBgJZE-TCkZgwauSBIM1rf_tt344YaIgP_na-Xt5U0azKjSFTmt9_YwgfweC_b_ndyZ6Nz1PtdHXaFpOO50C"
+    ]
+    @export_url "https://www.googleapis.com/drive/v3/files/#{@file_id}/export?mimeType=application/vnd.oasis.opendocument.text"
+    @invalid_export_url "https://www.googleapis.com/drive/v3/files/#{@file_id}/export?mimeType=application/invalid.mime.type"
+    @body <<80, 75, 3, 4, 20, 0, 0, 8, 0, 0, 10, 175, 132, 75, 94, 198, 50, 12, 39, 0, 0,
+    0, 39, 0, 0, 0, 8, 0, 0, 0, 109, 105, 109, 101, 116, 121, 112, 101, 97, 112,
+    112, 108, 105, 99, 97, 116, 105, 111, 110, 47>>
```

Comment:
What do these numbers mean?

Reply:
Google Drive API returns binary in body of requisition if an file return. Its is a representation of this binary